### DISCUSSION TITLE
Unable to find the ValheimToolerLauncher.exe

### DISCUSSION THREAD

**Discussion by ChatTheLad** [Astropilot/ValheimTooler#51] [General]
After download the Zip file and extract them out, I am not being able to find the ValheimToolerLauncher.exe anywhere whatsoever. Is it hidden away somewhere or something is missing?

**Comment by Astropilot** [comment]
Hi @ChatTheLad ,

Did you download the right zip file? It is the one named `ValheimTooler_1.11.0.zip`.
Also the launcher is sadly recognized as a virus (see why here: https://github.com/Astropilot/ValheimTooler/issues/34) by a lot of antivirus, so maybe your antivirus removed it as soon as you extracted the files. You may have to check the quarantine and make an exclusion.

Kind regards,
Astropilot
